domain-driven design/ValueObjects/ValueObjects/ValueObjects/EmailAddress.cs

```diff
@@ -0,0 +1,3 @@
+namespace ValueObjects.ValueObjects;
+
+public sealed record EmailAddress(string Address); //validation omitted for brevity
```

Comment:
We don't necessarily need to put these comments in the source code if it's simply for the purposes of the article. But it's fine to stay this time 😃